server/repository/src/migrations/v2_09_00/add_shipped_number_of_packs_to_invoice_line_legacy.rs

```diff
@@ -0,0 +1,52 @@
+use crate::migrations::{
+    templates::add_data_from_sync_buffer::{sync_buffer, SyncAction},
```

Comment:
We are not meant to be having any dependencies like this in migrations, i guess it would be fine in this case

Reply:
Because..?

Reply:
Blanket rule, to avoid us using 'repository' or 'service' layer which would have current data types that won't match migrations.

This case should be fine i guess because data type in migration/template won't change